src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,21 @@
+import { NavLink } from 'react-router-dom';
+import './Navigation.css';
+
+export const Navigation = () => {
+  return (
+    <nav className="navigation">
+      <NavLink className="nav-links" to="/">
+        Home
+      </NavLink>
+      <NavLink className="nav-links" to="/create">
+        Create
+      </NavLink>
+      <NavLink className="nav-links" to="/update">
+        Update
+      </NavLink>
+      <NavLink className="nav-links" to="/about">
+        About
+      </NavLink>
```

Comment:
As you can see we have some repetition `e.g. className="nav-links"`, so why don't we introduce some component to carry some basic style, but to be extendable as NavLink is?